### PR TITLE
New package: pdftk-2.02

### PR DESCRIPTION
**Pull request by thypon**
#1252

### CODE REVIEW

File: srcpkgs/pdftk/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'pdftk'
+pkgname=pdftk
+version=2.02
+revision=1
+wrksrc="${pkgname}-${version}-dist/pdftk"
+build_style=gnu-makefile
+hostmakedepends="unzip"
+makedepends="gcc-gcj"
```
Comment (xtraeme) [inline srcpkgs/pdftk/template]: - `gcc-gcj` should be in `hostmakedepends` instead.
- I really doubt this pkg needs `libgcj-devel` as runtime dependency.

File: srcpkgs/pdftk/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'pdftk'
+pkgname=pdftk
+version=2.02
+revision=1
+wrksrc="${pkgname}-${version}-dist/pdftk"
```
Comment (xtraeme) [inline srcpkgs/pdftk/template]: This is wrong, `wrksrc` should always be set to a top level directory in ${XBPS_BUILDDIR} (/builddir). Use this:

```
wrksrc=${pkgname}-${version}-dist
build_wrksrc=${pkgname}
```

File: srcpkgs/pdftk/template (hunk 1 above)
Comment (thypon) [inline srcpkgs/pdftk/template]: ops, my fault, i thought it was needed by makedepends.

**Comment by thypon** [comment]
Why the hell you rebase master....

**Comment by thypon** [comment]
Doing the same thing on my Worskstation with "./xbps-src pkg pdftk" doesn't fail.

**Comment by thypon** [comment]
the funny thing is that this ends the build on my pc:
```
[gattuso@razzi packages]$ tar tvf hostdir/binpkgs/pdftk/pdftk-2.02_1.x86_64.xbps 
-rw-r--r-- root/root      1252 2015-04-24 22:48 ./props.plist
-rw-r--r-- root/root      1097 2015-04-24 22:48 ./files.plist
-rw-r--r-- 0/0           23432 2015-04-24 22:48 ./usr/share/man/man1/pdftk.1
-rwxr-xr-x 0/0         2998976 2015-04-24 22:48 ./usr/bin/pdftk
```

**Comment by thypon** [comment]
I think that it fails on
```
gcj -Wno-all -fsource=1.3 -O2 --encoding=UTF-8 --classpath="/usr/share/java/libgcj.jar:/builddir/pdftk-2.02-dist/java:." -c java_lib.jar
```
because there is a per command timeout. This parti is very heavyweight and takes like 5/10 seconds on my 4770k.

**Pull request merged**